scripts/joinmarket-qt.py

```diff
@@ -261,21 +261,28 @@ def getSettingsFields(self, section, names):
             if name in config_types:
                 t = config_types[name]
                 if t == bool:
-                    qt = QCheckBox()
+                    sf = QCheckBox()
                     if val == 'testnet' or val.lower() == 'true':
-                        qt.setChecked(True)
+                        sf.setChecked(True)
                 elif t == 'amount':
-                    qt = BitcoinAmountEdit(val)
+                    sf = BitcoinAmountEdit(val)
                 elif not t:
                     continue
                 else:
-                    qt = QLineEdit(val)
+                    sf = QLineEdit(val)
                     if t == int:
-                        qt.setValidator(QIntValidator(0, 65535))
+                        if name in ["port", "rpc_port", "socks5_port",
+                                    "daemon_port"]:
+                            sf.setValidator(JMIntValidator(1, 65535))
+                        elif name == "tx_fees":
+                            # must account for both tx_fees settings type,
+                            # and we set upper limit well above default absurd
+                            # check just in case a high value is needed:
+                            sf.setValidator(JMIntValidator(1, 1000000))
```

Comment:
Why exactly 1000000 (1000 sat/vB) limit here? Just a guess that anything above that is most likely a huge mistake?

Reply:
Ahh, I didn't read comment above. :)